snuba/datasets/configuration/generic_metrics/entities/org_counters.yaml

```diff
@@ -0,0 +1,57 @@
+version: v1
+kind: entity
+name: org_metrics_counters
```

Comment:
I would make this `generic_org_metrics_counters`, otherwise the name will collide with the existing entity.